aat/src/resources/features/F-023/F-023.feature

```diff
@@ -0,0 +1,92 @@
+@F-023
+Feature: F-023: Print List Of Documents
```

Comment:
I really couldn't make sense of the operation name `Print List Of Documents`.
Eventually, this api operation is returning a printable document, not itself printing the list of documents.
Could we find out a better name use all around?

Reply:
updated to "Get Case as Printable Document".